index.js

```diff
@@ -64,6 +62,7 @@ const propTypes = {
   itemStyle: PropTypes.object,
   onSubmit: PropTypes.func,
   disableOverlay: PropTypes.bool,
+	modalVisible: PropTypes.string,
```

Comment:
Hi @syzer can you remove the space here? let's keep a consistent coding style. Thanks